ci-operator/config/openshift/coredns/openshift-coredns-master.yaml

```diff
@@ -23,7 +23,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.12
+    tag: golang-1.13
```

Comment:
There was some discussion about moving to 1.15 - can we not go to 1.15 here?

Reply:
We could jump to 1.15 here, I just noticed other components are still using 1.13 for the build_root. All of the other net edge components are on go 1.13 for build_root. Should this PR bump each component to go 1.15 in one go (no pun intended)?

Reply:
Actually, better idea, each of our components should be using `rhel-8-golang-1.15-openshift-4.6` for `build_root` (to match the image build bases). Ill amend this PR to accomplish that.

Reply:
@frobware I address your suggestion and re-pushed.

Reply:
> Actually, better idea, each of our components should be using `rhel-8-release-golang-1.15-openshift-4.6` for `build_root` (to match the image build bases). Ill amend this PR to accomplish that.
> xref https://coreos.slack.com/archives/CB95J6R4N/p1600696630055100

/lgtm

Based on the slack conversation.  Curious as to why we're the only component that currently references `rhel-8-release-golang-1.15-openshift-4.6 `.

Reply:
> Based on the slack conversation. Curious as to why we're the only component that currently references `rhel-8-release-golang-1.15-openshift-4.6 `.

I too am confused and nervous to see that other components are referencing older Go images, but let's do the right thing.

/lgtm